taipy/core/task/_task_manager.py

```diff
@@ -52,10 +53,16 @@ def _orchestrator(cls) -> Type[_AbstractOrchestrator]:
         return _OrchestratorFactory._build_orchestrator()
 
     @classmethod
-    def _set(cls, task: Task) -> None:
-        cls.__save_data_nodes(task.input.values())
-        cls.__save_data_nodes(task.output.values())
-        super()._set(task)
+    def _create(cls, task: Task) -> None:
+        for dn in itertools.chain(task.input.values(), task.output.values()):
+            _DataManagerFactory._build_manager()._repository._save(dn)
+        cls._repository._save(task)
```

Comment:
The `_create()` signature here is different from other `_create()` methods from other entity managers.

Other managers, scenario for example, accept a config as the input to create new entity.

However, `_TaskManager._create()` takes an instance of the Task entity.

This method is used by scenario creation, and in the tests. But we can consider removing it since it is not consistent.

Reply:
I believe it makes sense to align all the managers. But I am not sure about the semantics.

what do we expect for the `manager._create()` method? 
- To instantiate an entity in memory from various parameters?
- To save in the repository in-memory entity given as a parameter?
- To instantiate an entity and the nested entities?
- To instantiate an entity and save it in the repository?
- To instantiate an entity and the nested entities, save the entity and the nested ones.
- To save in the repository an in-memory entity and nested in-memory entities given as a parameter?

Reply:
After discussion, the difference is ok because there's no abstract signature for `_Manager._create()`.

The `_JobManager._create()` also receives a Task entity as the input